llvm/lib/Frontend/OpenMP/OMPIRBuilder.cpp

```diff
@@ -4817,11 +4817,39 @@ static Function *createOutlinedFunction(
     Builder.restoreIP(
         ArgAccessorFuncCB(Arg, Input, InputCopy, AllocaIP, Builder.saveIP()));
 
-    // Collect all the instructions
+    // Things like GEP's can come in the form of Constants, constants and
```

Comment:
I think it reads a little better this way, but maybe it's not what you're trying to say. Please ignore this suggestion if you don't agree with it.
```suggestion
    // Things like GEP's can come in the form of Constants. Constants and
```